NICE.Identity.Management/ClientApp/src/models/types.ts

```diff
@@ -37,6 +37,7 @@ export type WebsiteType = {
 	serviceId: number;
 	host: string;
 	environment: EnvironmentType;
+	service: ServiceType;
```

Comment:
Think I need to update this on mine. 🤕

Reply:
This will come in with the change I made to the API on https://github.com/nice-digital/identity/pull/128 and is also reflected in the change I made to the mockAPI in this pull request so you'd need those too.